worker/worker.go

```diff
@@ -325,6 +328,6 @@ func SetBinaryChecksum(checksum string) {
 }
 
 // NewJwtAuthorizationProvider creates a JwtAuthorizationProvider instance.
-func NewJwtAuthorizationProvider(privateKey []byte) auth.AuthorizationProvider {
+func NewJwtAuthorizationProvider(privateKey []byte) AuthorizationProvider {
```

Comment:
Can we change to NewAdminJwtAuthorizationProvider? I feel like it’s a better name and will remind people this may not be good for all scenarios.